src/plugins/legacy/medN4BiasCorrection/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+################################################################################
+#
+# medInria
+#
+# Copyright (c) INRIA 2013 - 2019. All rights reserved.
+# See LICENSE.txt for details.
+# 
+#  This software is distributed WITHOUT ANY WARRANTY; without even
+#  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#  PURPOSE.
+#
+################################################################################
+
+set(TARGET_NAME medN4BiasCorrectionPlugin)
+
+## #############################################################################
+## Setup version numbering
+## #############################################################################
+
+set(${TARGET_NAME}_VERSION ${MEDINRIA_VERSION})
+
+string(TOUPPER ${TARGET_NAME} TARGET_NAME_UP)
+add_definitions(-D${TARGET_NAME_UP}_VERSION="${${TARGET_NAME}_VERSION}")
+
+set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
+
+## #############################################################################
+## Resolve dependencies
+## #############################################################################
+
+find_package(dtk REQUIRED)
+include_directories(${dtk_INCLUDE_DIRS})
+
+find_package(ITK REQUIRED COMPONENTS ITKCommon ITKImageIntensity ITKBiasCorrection
+                                     ITKImageSources ITKThresholding)
+include(${ITK_USE_FILE})
+
+## #############################################################################
+## List Sources
+## #############################################################################
+
+list_source_files(${TARGET_NAME}
+  ${CMAKE_CURRENT_SOURCE_DIR}
+  )
+
+## #############################################################################
+## include directories
+## #############################################################################
+
+list_header_directories_to_include(${TARGET_NAME}
+  ${${TARGET_NAME}_HEADERS}
+  )
+
+include_directories(${${TARGET_NAME}_INCLUDE_DIRS}
+  ${MEDINRIA_INCLUDE_DIRS}
+  )
+
+## #############################################################################
+## add library
+## #############################################################################
+
+add_library(${TARGET_NAME} SHARED
+  ${${TARGET_NAME}_CFILES}
+  ${${TARGET_NAME}_QRC}
+  )
+
+## #############################################################################
+## Link
+## #############################################################################
+
+target_link_libraries(${TARGET_NAME}
+  ${QT_LIBRARIES}
```

Comment:
Is it enough ? no ITK libs ...
Is it done by line 36?

Reply:
The compilation does not ask anything here. Do you think you're going to have a problem here on Windows?